news/2023-01-04-featured-artist-track-updates-winter-2023.md

```diff
@@ -0,0 +1,378 @@
+</audio>
+
+### Imperial Circus Dead Decadence - Tensei
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/35%2FMOGARI%20-%20Shi%20e%20Fukeru%20Omoi%20wa%20Rikujoku%20Sura%20Kurai%2C%20Kanata%20no%20Sei%20o%20Aisuru%20Tame%20ni%20Inoti%20o%20Tataeru%2FImperial%20Circus%20Dead%20Decadence%20-%20Tensei.mp3" type="audio/mpeg">
+</audio>
+
+## [Kurokotei](https://osu.ppy.sh/beatmaps/artists/58)
+
+[![](https://assets.ppy.sh/artists/58/header.jpg)](https://osu.ppy.sh/beatmaps/artists/58)
+
+**Kurokotei** and **Sennzai** teamed up to create an original song for osu! World Cup's Semi Finals Tiebreaker! It even came jam-packed with background art created by fellow Featured Artist [Se-U-Ra](https://osu.ppy.sh/beatmaps/artists/112). Can musicians draw? Find out for yourself:
+
+### Kurokotei feat. Sennzai - escape (the looking-glass, and what alice found there)
+
+Escape down the rabbithole and into [this map used as OWC's Semi Finals Tiebreaker](https://osu.ppy.sh/beatmapsets/1880652) hosted by [Realazy](https://osu.ppy.sh/users/918297)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/58/Songs/Kurokotei%20feat.%20Sennzai%20-%20escape%20(the%20looking-glass%2C%20and%20what%20alice%20found%20there).mp3" type="audio/mpeg">
+</audio>
+
+## [Rabbit House](https://osu.ppy.sh/beatmaps/artists/242)
+
+[![](https://assets.ppy.sh/artists/242/header.jpg)](https://osu.ppy.sh/beatmaps/artists/242)
+
+**Rabbit House** (a.k.a. the dude who mapped [FREEDOM DiVE](https://osu.ppy.sh/beatmapsets/39804)) helped out with the osu! World Cup too with their song *Sanctuary*.
+
+### Rabbit House - Sanctuary
+
+Explore [this beatmap](https://osu.ppy.sh/beatmapsets/1889094) hosted by [funny](https://osu.ppy.sh/users/8473573) and used in OWC's Grand Finals HR4 slot.
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/242/Songs/Rabbit%20House%20-%20Sanctuary.mp3" type="audio/mpeg">
+</audio>
+
+## [Raimukun](https://osu.ppy.sh/beatmaps/artists/247)
+
+[![](https://assets.ppy.sh/artists/247/header.jpg)](https://osu.ppy.sh/beatmaps/artists/247)
+
+Another artist, another OWC osu!original. **Raimukun** showcases an endless string of piano notes in their newest tune *Myths Orbis*!
+
+### Raimukun - Mythis Orbis
+
+Muscle spasm your way through [this map used as OWC's Grand Finals NM2](https://osu.ppy.sh/beatmapsets/1889095) hosted by [knowledgeking](https://osu.ppy.sh/users/8022517)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/247/Songs/Raimukun%20-%20Myths%20Orbis.mp3" type="audio/mpeg">
+</audio>
+
+## [Redside](https://osu.ppy.sh/beatmaps/artists/184)
+
+[![](https://assets.ppy.sh/artists/184/header.jpg)](https://osu.ppy.sh/beatmaps/artists/184)
+
+You know the drill. osu! original here from last year's osu! World Cup. This one's from the Quarter Finals though, so it might be a tad more playable than other originals!
```

Comment:
```suggestion
You know the drill. osu! original here from last year's osu! World Cup. This one's from the Quarterfinals though, so it might be a tad more playable than other originals!
```